src/moscot/problems/space/_mixins.py

```diff
@@ -562,6 +604,38 @@ def cell_transition(  # type: ignore[misc]
             key_added=key_added,
         )
 
+    def celltype_mapping(
```

Comment:
missing docstrings.
also introduce a variable "cell_transition_kwargs".

Reply:
"cell_transition_kwargs" should be default types.MappingProxyType({}). Raise error if `cell_transition_kwargs` is not an empty `dict` although `cell_transition` is not used